app/templating/AiHelper.scala

```diff
@@ -8,7 +8,7 @@ import lila.app.ui.ScalatagsTemplate._
 trait AiHelper { self: I18nHelper =>
 
   def aiName(level: Int, withRating: Boolean = true)(implicit lang: Lang): String = {
-    val name = trans.aiNameLevelAiLevel.txt("Engine", level)
+    val name = trans.aiNameLevelAiLevel.txt("YaneuraOu V7.00", level)
```

Comment:
I'm not sure that names or version numbers usually go in translation contexts.

Reply:
On crowdin `aiNameLevelAiLevel` is translated as `%1$s level %2$s` where `%1$s` will be `YaneuraOu V7.00`. So I don't think this should be a problem right